frontend/src/component/common/Search/Search.tsx

```diff
@@ -27,9 +35,26 @@ interface ISearchProps {
     expandable?: boolean;
 }
 
+const StyledPaper = styled(Paper)(({ theme }) => ({
```

Comment:
is this a copy paste? can we extract this component and share?

Reply:
Yes